### PR TITLE
JPN-489 Filter out bots from top contributors using user_id

### PR DESCRIPTION
**Pull request by adamkarminski**
https://wikia-inc.atlassian.net/browse/JPN-489

This approach makes use of the list of global bots we already have.

@Wikia/spitfires

### CODE REVIEW

**Comment by lukaszkonieczny** [comment]
LGTM

File: extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php [hunk 1]
```diff
@@ -63,31 +62,30 @@ function () use ( $limit, $weekly, $onlyAdmins, $botIds ) {
 				}
 
 				$sqlData = ( new WikiaSQL() )
-					->SELECT( 'user_name, user_id, ug_group, count(rev_id) AS revision_count' )
+					->SELECT( 'rev_user_text, rev_user, count(rev_id) AS revision_count' )
 					->FROM ( 'revision FORCE INDEX (user_timestamp)' )
-					->LEFT_JOIN( $wgExternalSharedDB . '_' . $wgDBcluster . '.user' )->ON( '(rev_user <> 0) AND (user_id = rev_user)' )
 					->LEFT_JOIN( 'user_groups ON (user_id = ug_user)' )
```
Comment (lukaszkonieczny) [inline extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php]: should be `rev_user = ug_user`

File: extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php [hunk 2]
```diff
@@ -65,27 +62,30 @@ function () use ( $limit, $weekly, $onlyAdmins ) {
...
```
Comment (lukaszkonieczny) [inline extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php]: should be `rev_user = ug_user`

Comment (lukaszkonieczny) [inline extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php]: and probably we need this `LEFT JOIN` only to get admins

File: extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php [hunk 3]
```diff
@@ -104,12 +107,12 @@ function () {
 				$db = wfGetDB( DB_SLAVE, [], $wgExternalSharedDB );
```
Comment (lukaszkonieczny) [inline extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php:109]: is it possible that also local bots exist?

**Comment by adamkarminski** [comment]
LGTM

**Pull request merged**